markdownhighlighter.cpp

```diff
@@ -2196,6 +2196,23 @@ bool MarkdownHighlighter::isPosInACodeSpan(int blockNumber, int position) const
     }) != rangeList.cend();
 }
 
+QPair<int, int> MarkdownHighlighter::codeSpanRange(int blockNumber, int position) const
+{
+    const QVector<InlineRange> rangeList = _ranges.value(blockNumber);
+    const auto it = std::find_if(rangeList.cbegin(), rangeList.cend(),
+                                     [position](const InlineRange& range){
+        if (position > range.begin && position < range.end && range.type == RangeType::CodeSpan)
+            return true;
+        return false;
+    });
+
+    if (it == rangeList.cend()) {
+        return {};
```

Comment:
Should this not return - 1, - 1 instead?

Reply:
@fnkabit ?

Reply:
this isn't resolved yet, right?